### ISSUE TITLE
Missing number of ignored statements in Install SQL Patches message 

### ISSUE THREAD

**Issue by scottcwilson** [zencart/zencart#3951]
<img width="772" alt="Screen Shot 2020-09-10 at 4 43 08 PM" src="https://user-images.githubusercontent.com/4391638/92813334-fc2dff80-f38e-11ea-81e2-314a3b43e15b.png">

**Comment by scottcwilson** [comment]
It looks as though this has been fixed in 1.5.8 (or earlier).

**Comment by scottcwilson** [comment]
```
grep ERROR_UPLOADQUERY_IGNORED `ff`
./includes/languages/english/lang.sqlpatch.php:    'ERROR_UPLOADQUERY_IGNORED' => 'Note: %u statements ignored via file upload. See database table "upgrade_exceptions" for additional details.',
./sqlpatch.php:          $messageStack->add(sprintf(ERROR_UPLOADQUERY_IGNORED, (int)$query_results['ignored']), 'caution');
``
